rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1635,119 +1663,133 @@ private Optional<Federation> getFederationFromPublicKey(BtcECKey federatorPublic
         return Optional.empty();
     }
 
+    private void logMissingSignatures(BtcTransaction btcTx, Keccak256 rskTxHash, int neededSignatures, int federationSize) {
+        int missingSignatures = BridgeUtils.countMissingSignatures(btcContext, btcTx);
+        int signaturesCount = neededSignatures - missingSignatures;
+
+        logger.debug("Tx {} not yet fully signed. Requires {}/{} signatures but has {}",
```

Comment:
maybe this can be made at the `INFO` level. It seems interesting/important to me, wdyt?

Reply:
i think it may add lot of logs. Have in mind that it will appear every time one fed signs a tx

Reply:
I think it should not add that many. It should only appear `n-1` times per tx. Notice with the pegout signed cache in place, addSignature should only get triggered once. I would find this info interesting as a fed node operator